doc/user/content/get-started/isolation-level.md

```diff
@@ -119,7 +119,8 @@ linearizable transactions, then you should downgrade to the Serializable isolati
 
 Strict Serializable provides stronger consistency guarantees but may have slower reads than Serializable. This is
 because Strict Serializable may need to wait for writes to propagate through materialized views and indexes, while
-Serializable does not.
+Serializable does not. For details about this behavior, consult the documentation
+on [logical timestamp selection](/sql/functions/now_and_mz_now#logical-timestamp-selection).
```

Comment:
I feel like you have a funny rebase in here?

Reply:
I do!